packages/uswds/src/Templates/WrapIfAdditionalTemplate.tsx

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import {
+  WrapIfAdditionalTemplateProps,
+  FormContextType,
+  RJSFSchema,
+  StrictRJSFSchema,
+} from "@rjsf/utils";
+
+/** The `WrapIfAdditional` component is used by the `FieldTemplate` to wrap automatically added additional properties.
+ * This component prevents the additional property buttons from appearing when the `additionalProperties` keyword is
+ * set to false.
+ *
+ * @param props - The `WrapIfAdditionalProps` for the component
+ */
+const WrapIfAdditionalTemplate = <
+  T = any,
+  S extends StrictRJSFSchema = RJSFSchema,
+  F extends FormContextType = any,
+>({
+  children,
+  classNames,
+  style,
+  disabled,
+  id,
+  label,
+  onDropPropertyClick,
+  onKeyChange,
+  readonly,
+  required,
+  registry,
+  schema,
+  uiSchema,
+}: WrapIfAdditionalTemplateProps<T, S, F>) => {
```

Comment:
We prefer functions to fat-arrows
```suggestion
export default WrapIfAdditionalTemplate<
  T = any,
  S extends StrictRJSFSchema = RJSFSchema,
  F extends FormContextType = any,
>({
  children,
  classNames,
  style,
  disabled,
  id,
  label,
  onDropPropertyClick,
  onKeyChange,
  readonly,
  required,
  registry,
  schema,
  uiSchema,
}: WrapIfAdditionalTemplateProps<T, S, F>) {
```